cmd/stack_create.go

```diff
@@ -108,10 +106,12 @@ The stack name must start with a lowercase letter, and can contain only lowercas
 
 			//moving out the stack which we need
 			if config.Dryrun {
-				Info.logf("Dry Run -Skipping moving out of stack: %s from %s", projectType, filepath.Join(stack, "stacks-master", config.copy))
+				Info.logf("Dry Run -Skipping moving out of stack: %s from %s", stackTempDir, filepath.Join(stack, "stacks-master", config.copy))
 
 			} else {
-				err = os.Rename(filepath.Join(stack, "stacks-master", config.copy), projectType)
+				stackTempDir = ".temp-" + stackTempDir + "-" + currentTime
```

Comment:
Just a small suggestion, this line should be moved up to where you first define the `stackTempDir` variable (line 90)

Reply:
I'm setting the value of `stackTempDir` variable in else case because we want to set the value of `stackTempDir` if the user has not selected dry run mode.

If the user is running in the dry run mode, then we want the value of `stackTempDir` to be the existing stack name which the user has given in copy flag as I'm using that in (line 109).

Reply:
I don't understand why you want to print something different in dryrun mode. Shouldn't you print what will actually happen?

Reply:
When we unzip a particular stack e.g. nodejs which user wants, it is in the structure `stacks-master/incubator/nodejs`. In (line 109) I'm printing that I'm moving out the stack which user want so when the user is running dry run mode, I want to print properly the name of the stack which I'm moving out, not the name of the temporary directory.

Reply:
ok thanks. I'll approve and merge